pipelines/training.py

```diff
@@ -780,6 +782,21 @@ def everything(
             )
         train_s3_datadir = generate_dataset_op.outputs['s3_datadir']
 
+    ood_classification_op = ood_classification_step(
```

Comment:
This should be conditional and only enabled in pipelines with calibrate, otherwise it will affect unrelated projects.